test/conformance/event/event_adapter_opencl.match

```diff
@@ -0,0 +1,3 @@
+# Note: This file is only for use with cts_exe.py
```

Comment:
There are two programs now that handle match files; the older matching logic and cts_exe.py. When I updated the files, I did so in a way incompatible with the old match checker thing. Figured I'd add a note calling that out.

Although, happy to remove it now that we've transitioned to using cts_exe.py.

Reply:
Ah okay make sense, I think it is better to keep it for now to avoid confusion then with the old matching logic. might be worth also calling out explicitly that this is incompatible with the old matching system. (doesn't have strong opinions about this)

Reply:
https://github.com/oneapi-src/unified-runtime/pull/2285 Decided to go hard in the other direction. Drop support for the older matching logic and make our testing files look neater. Not sure how popular a change this will be though.